src/main/java/core/basesyntax/operations/PurchaseOperation.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseOperation implements OperationHandler {
+    private final Storage storage;

```

Comment:
The `PurchaseOperation` class should not have a direct dependency on `Storage`. Instead, consider using dependency injection to pass the necessary data or services to this class.